src_modules/voice_commands/index.ts

```diff
@@ -142,18 +143,30 @@ export default class VoiceCommandsModule extends Module {
         if (oldState.id !== this.bot.client!.user!.id || newState.id !== this.bot.client!.user!.id) {
             return;
         }
-
-        if (oldState.channel === null && newState.channel !== null) {
+    
+        if (oldState.channel === null && newState.channel !== null && !newState.member.user.bot) {
```

Comment:
Leave this if block as is, and move all the changes to a new if block above this one.

Reply:
Move this if block above the early return above.